stress-job.groovy

```diff
@@ -5,6 +5,7 @@ pipelineJob("Stress-job") {
         stringParam('BRANCH', '', 'James branch')
         stringParam('SIMULATION', '', 'Gatling simulation')
         stringParam('GITHUB_SIMULATIONS', '', 'Github repository containing gatling simulations')
+        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), swift or s3')
```

Comment:
I don't understand what this "reference" blobstore is

Reply:
I cannot find a descent name: the one we have built for performance objectives for the secret project

Reply:
Why not just `BLOB_STORE_TARGET` or something similar? Instead of `PROFILE`?

Reply:
https://github.com/linagora/james-project/pull/3334#issuecomment-619945296

Reply:
Let's say then that it's not about "blob store" but really "profile".
That means there are arbitrary values in the list that each one describe a setup.
`reference` is: component A, B, C + pre-provisioned data
`s3` is: component A, B, C + OVH S3
That would make sense, IMO, but needs a bit more documentation

Reply:
Sure, where?